src/discovery/azure/Akka.Discovery.Azure/Actors/PruneActor.cs

```diff
@@ -0,0 +1,170 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="PruneActor.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Cluster;
+using Akka.Event;
+using Akka.Util.Internal;
+using DotNetty.Common.Utilities;
+
+namespace Akka.Discovery.Azure.Actors
+{
+    internal class PruneActor: UntypedActor, IWithTimers
```

Comment:
Needs some XML-DOC - should explain how this actor gets used.

Reply:
Will add in a separate PR